tests/conftest.py

```diff
@@ -5007,12 +5007,17 @@ def profiler_store_name() -> str:
 
 
 @pytest.fixture(scope="function")
-def profiler_config(profiler_name: str) -> RuleBasedProfilerConfig:
+def profiler_config_with_placeholder_args(
+    profiler_name: str,
+) -> RuleBasedProfilerConfig:
```

Comment:
Please add a docstring to clarify what you mean by placeholder_args (and how this differs from a standard use-case)

Reply:
@cdkini I added a docstring.  Thanks.